config/helm/chart/default/templates/Common/csi/daemonset.yaml

```diff
@@ -87,11 +87,11 @@ spec:
           protocol: TCP
         resources:
           limits:
-            cpu: {{ default "300m" ((.Values.csi).limits).cpu }}
-            memory: {{ default "100Mi" ((.Values.csi).limits).memory }}
+            cpu: {{ default "100m" ((.Values.csi).limits).cpu }}
+            memory: {{ default "50Mi" ((.Values.csi).limits).memory }}
           requests:
-            cpu: {{ default "300m" ((.Values.csi).requests).cpu }}
-            memory: {{ default "100Mi" ((.Values.csi).requests).memory }}
+            cpu: {{ default "100m" ((.Values.csi).requests).cpu }}
+            memory: {{ default "50Mi" ((.Values.csi).requests).memory }}
```

Comment:
we should either have a separate requests/limits section for these, or just static ones as the server part should have a stable memory/cpu usage on different environments